test/functional/ui/wildmode_spec.lua

```diff
@@ -64,3 +65,77 @@ describe('command line completion', function()
     ]])
   end)
 end)
+
+describe('External command line completion', function()
+  local screen
+  local items, selected
+
+  before_each(function()
+    clear()
+    screen = Screen.new(25, 5)
+    screen:attach({rgb=true, wildmenu_external=true})
+    screen:set_on_event_handler(function(name, data)
+      if name == "wildmenu" then
+        items = data
+      elseif name == "wildmenu_select" then
+        selected = data[1]
+      elseif name == "wildmenu_hide" then
+        items = nil
+      end
+    end)
```

Comment:
The wildmenu events may be best handled in screen.lua via functions Screen:_handle_wildmenu_select/Screen:_handle_wildmenu_hide etc as otherwise, any wildmenu events in other tests will make the test fail (as there will be no handler for the event).